find_target_name_from_header.py

```diff
@@ -277,67 +255,23 @@ def add_separation_between_pointing_current_position(coords,simbad_dico):
         coords_current = SkyCoord(coords_current_str,frame=ICRS,unit=(u.hourangle,u.deg))
         sep_pointing_current = coords.separation(coords_current).to(u.arcsec).value
         simbad_dico['simbad_separation_RADEC_current']=sep_pointing_current
-        print('Distance between the current star position and pointing position: {0:.1f}arcsec'.format(simbad_dico['simbad_separation_RADEC_current']))
+        if verbose:
+            print('Distance between the current star position and pointing position: {0:.1f}arcsec'.format(simbad_dico['simbad_separation_RADEC_current']))
     return simbad_dico
 
-def is_moving_object(header):
```

Comment:
Same comment as the first one. I'd need to keep those lines here

Reply:
See my first comment